### ISSUE TITLE
LiveReload snippet not being inserted

### ISSUE THREAD

**Issue by callumlocke** [yeoman/generator-webapp_DEPRECATED#77]
Here's what I did:

```
npm install -g yo grunt-cli bower
npm install generator-webapp
yo webapp
  # Bootstrap for Sass - YES
  # RequireJS - NO
  # (bower install & npm install completed successfully)
grunt server
```

But then when I edit `app/index.html`, Chrome doesn't refresh. When I view source, I can see there is no LR snippet.

This is an out-of-the-box `yo webapp`, generated straight after updating everything, and the Gruntfile.js looks normal ([pasted here](http://pastie.org/8000043#4,67)).

I just tried running `grunt server` on another project which I scaffolded a couple of weeks ago, and the LR middleware still works correctly on that one. The problem seems to be limited to all new projects I generate with `yo webapp`.

I'm not sure what to try next to troubleshoot this... Any suggestions?

**Comment by passy** [comment]
Duplicate of #63, fixed with #67. There hasn't been a release since then, but will probably happen at some point this week. You can also use the version in master to get it now. :)

**Comment by callumlocke** [comment]
Ah missed that. I'll use master, thanks
